OlexanderHramatovych/HW6/6-2.py

```diff
@@ -0,0 +1,9 @@
+
+# creating a loop which checks the login
+while True:
```

Comment:
good job
but look at this implementation 
user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)